modules/perception/camera/lib/traffic_light/detector/recognition/classify.cc

```diff
@@ -95,10 +96,9 @@ void ClassifyBySimple::Init(
         << input_reshape[net_inputs_[0]][2] << ", "
         << input_reshape[net_inputs_[0]][3];
 
-  // TODO(zhangxiao23): Init traffic light recognition network
-  // if (!rt_net_->Init(input_reshape)) {
-  //   AINFO << "net init fail.";
-  // }
+  if (!rt_net_->Init(input_reshape)) {
+    AINFO << "net init fail.";
```

Comment:
Here, if `net` init failure is considered unexpected, you'd better use  `AWARNING` to arise attention. And determine if early return is needed in such cases.